advanced_samples/Dataview_Wind/Jupyter/README.md

```diff
@@ -0,0 +1,74 @@
+# OCS Dataview Data Analysis Sample using Jupyter
+
+**Version:** 1.0.0
+
+[![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/PI-System/PIWebAPI_Data_Analysis?branchName=master)](https://dev.azure.com/osieng/engineering/_build?definitionId=1644&branchName=master)
+
+The sample code in this folder demonstrates how to utilize the OCS Dataviews to do some basic data analysis using Python Jupyter Notebook. In order to run this sample, you need to have [Python](https://www.python.org/downloads/) installed.  
+
+## About the Sample
+
+This sample is intended to show you how you can use an OCS Dataview to bring your data from OCS into Jupyter and easily into Pandas to do machine learning.  Using an OCS Dataview allows you to configure the data you want, in the way you want it, and not have to do as much client side processing of the data.  This is a basic sample that shows you a possible way to do machine learning and a possible outcome of looking at the data.  This is not a guide to Pandas or machine learning.
+
+## Background and Problem
+
+### Background
+
+Wind turbines operate by capturing kinetic enrgy of wind to turn rotor blades that run a generator.  Governed by Betz's law they can only capture 59.3% of kinetic enrgy from wind.  This is important, because after a certain limit higher winds do not create additional power.  At a certain wind speed, the power actually decreases. 
+
+### Problem Statement
+
+Can we predict what the power output of these wind turbines is based on weather forecast data.
+
+### Data Overview
+
+The data we are using is available in our BulkUpload SampleCollections [folder](https://github.com/osisoft/OSI-Samples-OCS/tree/master/advanced_samples/Bulk_Upload/SampleCollections/DataviewWind).  The steps to upload this data are included in that folder.  This sample also skips over the exercise of creating the dataview, this is included in the data that you can upload.  To see how to create a Dataview programmatically, please look at our other sample [here](https://github.com/osisoft/OSI-Samples-OCS/tree/master/basic_samples/DataViews). 
+
+This sample data is similar to what you might have from a site where the data has unexpected values and not every turbine behaves exactly the same.  The data is only for 1 day.  Repeating this exercise but using more days of data would give you a better prediction.    
+
+
+
+## Getting Started
+
+- Clone the GitHub repository
+- Install the required modules by running the following command in the terminal : `pip install -r requirements.txt`
+- If running the tests, install the required modules by running the following command in the terminal : `pip install -r test-requirements.txt`
+
+### Setting up the OCS connection
+
+- In the `Jupyter` folder, populate the values of `config.ini` with your own system configuration.
```

Comment:
`config.ini` should be renamed to `config.placeholder.ini` and this README should be updated to mention that this file needs to be renamed.